File: CS/04_Shapes/GetTextFrameSizeOfShape.cs
using System;
using System.Drawing;
using System.IO;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;

namespace GetTextFrameSizeOfShape
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create a PPT document
            Presentation presentation = new Presentation();
            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\GetTextFrameSizeOfShape.pptx");

            //Get the first slide
            ISlide slide = presentation.Slides[0];

            StringBuilder sb = new StringBuilder();

            // Iterate the shapes in the slide
            for (int i=0;i<slide.Shapes.Count;i++)
            {

                IAutoShape autoShape = slide.Shapes[i] as IAutoShape;
                SizeF size =  autoShape.TextFrame.GetTextSize();
                sb.AppendLine("The size of text frame in shape" + i + ", width:" + size.Width + " height:" +size.Height);
            }

            File.WriteAllText("GetTextFrameSizeOfShape.txt", sb.ToString());

            System.Diagnostics.Process.Start("GetTextFrameSizeOfShape.txt");

            presentation.Dispose();
        }
    }
}

File: VB/04_Shapes/GetTextFrameSizeOfShape.vb
Imports System.IO
Imports System.Text
Imports Spire.Presentation

Namespace GetTextFrameSizeOfShape
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create a PPT document
			Dim presentation As New Presentation()
			presentation.LoadFromFile("..\..\..\..\..\..\Data\GetTextFrameSizeOfShape.pptx")

			'Get the first slide
			Dim slide As ISlide = presentation.Slides(0)

			Dim sb As New StringBuilder()

			' Iterate the shapes in the slide
			For i As Integer = 0 To slide.Shapes.Count - 1

				Dim autoShape As IAutoShape = TryCast(slide.Shapes(i), IAutoShape)
				Dim size As SizeF = autoShape.TextFrame.GetTextSize()
				sb.AppendLine("The size of text frame in shape" & i & ", width:" & size.Width & " height:" & size.Height)
			Next i

			File.WriteAllText("GetTextFrameSizeOfShape.txt", sb.ToString())

			Process.Start("GetTextFrameSizeOfShape.txt")

			presentation.Dispose()
		End Sub
	End Class
End Namespace